Add guideline on how to contribute to the meetup and extend the contact information (#31)
WebEngDUS/WebEngDUS: README.md

Everything you need to know about how we run the *Web Engineering DUS Meetup*.

## Getting involved

> If you want to get involved, please check out our [Contribution Guide](./CONTRIBUTING.md).

Reviewer: ```suggestion
If you would like to get involved, please check out our [Contribution Guide](./CONTRIBUTING.md).
```

* [WebEngDUS on GitHub](https://github.com/WebEngDUS)
* [WebEngDUS on Speaker Deck](https://speakerdeck.com/webengdus)

## Organizers and contact

> Having a question or feedback for us?

Reviewer: ```suggestion
Have a question or feedback for us?
```

* [Open an issue in our GitHub issue tracker](https://github.com/WebEngDUS/WebEngDUS/issues/new)
* Speak to us at one of the local events

> * Write us a message via email or social media

Reviewer: ```suggestion
* Write us an email or contact us on social media
```
